shared-libs/transitions/src/transitions/registration.js

```diff
@@ -125,6 +126,49 @@ const getDaysSinceDOB = doc => {
   return findFirstDefinedValue(doc, fields);
 };
 
+const getLMPDateFromParts = doc => {
+  //basic validations will be done in the app_settings.registrations
+  return getDateFromParts(doc.fields.lmpYYYY, doc.fields.lmpMM, doc.fields.lmpDD);
+};
+
+const getLMPDateFromString = dateString => {
```

Comment:
@binokaryg I think all of these Bikram Sambat date manipulation should be placed in the `bikram-sambat` library to be reused, instead of inlined in the registration transition.

Reply:
Thanks for the early feedback.

Can you please specify which ones can be moved to the `bikram-sambat` library? The manipulation here is not specific to the Bikram Sambat calendar.

Reply:
The PR is now ready for review. Most of the work is done in `smsParser` and `validation` now.